google/cloud/forseti/notifier/notifier.py

```diff
@@ -136,17 +136,23 @@ def run(inventory_index_id, progress_queue, service_config=None):
     global_configs = service_config.get_global_config()
     notifier_configs = service_config.get_notifier_config()
 
+    scanner_index_id = None
```

Comment:
We should also support the use case if someone wants to run the notifier directly on the scanner_index_id.
```
forseti notifier run --scanner_index_id <scanner index id>
```
This okay to do in a follow-up.  Can you please open an issue to track this?

Reply:
Done. Issue #1405 .